ui/app/css/variables/typography.scss

```diff
@@ -0,0 +1,121 @@
+$fa-font-path: 'fonts/fontawesome';
+
+@import '../../../../node_modules/@fortawesome/fontawesome-free/scss/fontawesome';
+@import '../../../../node_modules/@fortawesome/fontawesome-free/scss/solid';
+@import '../../../../node_modules/@fortawesome/fontawesome-free/scss/regular';
+
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 100;
+  src: local('Roboto Thin'), local('Roboto-Thin'), url('fonts/Roboto/Roboto-Thin.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 300;
+  src: local('Roboto Light'), local('Roboto-Light'), url('fonts/Roboto/Roboto-Light.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Roboto'), local('Roboto-Regular'), url('fonts/Roboto/Roboto-Regular.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 500;
+  src: local('Roboto Medium'), local('Roboto-Medium'), url('fonts/Roboto/Roboto-Medium.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 700;
+  src: local('Roboto Bold'), local('Roboto-Bold'), url('fonts/Roboto/Roboto-Bold.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 900;
+  src: local('Roboto Black'), local('Roboto-Black'), url('fonts/Roboto/Roboto-Black.ttf') format('truetype');
+}
+
+$font-family: Roboto, Helvetica, Arial, sans-serif;
+
+// Typography
+%H1 {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 2.5rem;
+  font-family: $font-family;
+}
+
+%H2 {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 2rem;
+  font-family: $font-family;
+}
+
+%H3 {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1.5rem;
+  font-family: $font-family;
+}
+
+%H4 {
+  font-style: normal;
```

Comment:
I guess the same goes for `font-family` as well

Reply:
For the most part I tried to make sure that the changes I made resulted in very close to the original styling, but because of the nature of CSS, it's possible that due to cascades setting font-style: normal; might override a cascaded font-style from a parent. I think a UI review is probably in order as you mentioned in you review comment. All the typography styles in the design system specify font-style which is why I specified it here.